source/crud/tests/unified/create-null-ids.yml

```diff
@@ -0,0 +1,288 @@
+description: CRUD ID Type Tests
+schemaVersion: '1.0'
+createEntities:
+  - client:
+      id: client
+      observeEvents:
+        - commandStartedEvent
+  - database:
+      id: database
+      client: client
+      databaseName: crud_id
+  - collection:
+      id: collection
+      database: database
+      collectionName: type_tests
```

Comment:
I realize this file was generated, but you can still use YAML anchors for the database and collection names for consistency with other files. Hopefully it's just a quick search/replace operation.

I understand that won't handle the full namespace string for client bulkWrite, but it should remove repetition elsewhere.

Reply:
Added anchors, and also used some for the input data